src/app/_utils/capitalise.ts

```diff
@@ -0,0 +1,5 @@
+export default function Capitalise(word: string) {
+  if (!word || word.length === 0) return word
+
+  return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()
+}
```

Comment:
Shall we use CSS? `text-transform: capitalize;` || Tailwind [>>](https://tailwindcss.com/docs/text-transform)

Reply:
oh cool, didn't know Tailwind had that